src/async.js

```diff
@@ -28,6 +29,12 @@ Async.prototype.setScheduler = function(fn) {
     return prev;
 };
 
+// set the batchsize for processing queue
+Async.prototype.setBatchSize = function(batchSize) {
+    ASSERT(batchSize > 0);
```

Comment:
Agreed that batchSize <= 0 is invalid, but `batchSize = undefined` should also be supported (i.e. in case infra team wants to use this copy of bluebird but doesn't want to bother with hooking up an external dispatcher and using batching). `batchSize=undefined` should be the default, and Hadron should have to call setBatchSize() to get a non-default batch size.